### PR TITLE
Computer updates

### PR DESCRIPTION
**Pull request by ghost**
:cl: Xhuis
add: All computers now have sounds. Try them out!
/:cl:

Updates terminals by adding sounds and some visual updates, such as the camera console flashing static when you switch cameras. This is WIP and will include all terminals eventually. [Demo here](https://dl.dropboxusercontent.com/u/116194515/development_media/terminal_sounds.mp4).

### CODE REVIEW

File: code/game/machinery/computer/camera.dm [hunk 1]
```diff
@@ -100,12 +104,16 @@
...
 
+		playsound(src, "terminal_type", 25, 0)
+		playsound(src, 'sound/machines/terminal_prompt_confirm.ogg', 25, 0)
 		if(isAI(user))
 			var/mob/living/silicon/ai/A = user
 			A.eyeobj.setLoc(get_turf(C))
 			A.client.eye = A.eyeobj
 		else
 			user.reset_perspective(C)
+			user.overlay_fullscreen("flash", /obj/screen/fullscreen/flash/noise)
```
Comment (Xhuis) [inline code/game/machinery/computer/camera.dm]: It doesn't seem to exist.

Comment (Xhuis) [inline code/game/machinery/computer/camera.dm]: Doesn't seem really in the scope of PR, but I'll do it. I blame you if oranges gets angry at me for not atomizing again.

**Comment by GunHog** [comment]
Sharpens the future-retro theme!

**Comment by Xhuis** [comment]
* Renamed the "noise" screen effect o "static" at Remie's insistence.
* Added sounds for AI restoration, camera, ID modifier, and cloning consoles.

**Comment by Chiefwaffles** [comment]
I'm sure I'm 100% wrong, but it feels like the effect would be better without the fade in/out?

**Comment by Xhuis** [comment]
It doesn't fade in - it just pops up. The fade-out is sadly an unavoidable part of fullscreen overlay code.

**Comment by Xhuis** [comment]
Damn it, I said not to merge it. This wasn't done. Too late now.

**Comment by GunHog** [comment]
You can just patch it.

**Pull request merged**